sdk/modelsrepository/Azure.Iot.ModelsRepository/src/Fetchers/FetchResult.cs

```diff
@@ -15,7 +15,7 @@ internal class FetchResult
         public string Path { get; set; }
 
         public bool FromExpanded => Path.EndsWith(
-            ModelRepositoryConstants.ExpandedJsonFileExtension,
+            ModelsRepositoryConstants.ExpandedJsonFileExtension,
             System.StringComparison.InvariantCultureIgnoreCase);
```

Comment:
Prefer move `System` to `using System;` above.

Reply:
Done in recently opened PR https://github.com/Azure/azure-sdk-for-net/pull/19072